src/sentry/demo/middleware.py

```diff
@@ -0,0 +1,30 @@
+from django.conf import settings
+
+from sentry.models import OrganizationMember
+from sentry.utils import auth
+
+
+class DemoMiddleware:
+    # automatically log in logged out users when they land
+    # on organization pages
+    def process_view(self, request, view_func, view_args, view_kwargs):
+        if not settings.DEMO_MODE:
+            return
```

Comment:
I would raise an exception here so that the application fails to start as an additional precaution.